src/core/ext/filters/client_channel/client_channel.h

```diff
@@ -382,6 +384,18 @@ class ClientChannel {
 class ClientChannel::LoadBalancedCall
     : public InternallyRefCounted<LoadBalancedCall, kUnrefCallDtor> {
  public:
+  class LbCallState : public LoadBalancingPolicy::CallState {
+   public:
+    explicit LbCallState(LoadBalancedCall* lb_call) : lb_call_(lb_call) {}
+
+    void* Alloc(size_t size) override { return lb_call_->arena_->Alloc(size); }
+
+    absl::string_view GetCallAttribute(UniqueTypeName type);
```

Comment:
It's unclear to me what this method does: can we have a comment

Reply:
Good idea, done.